travis-build.sh

```diff
@@ -33,11 +33,18 @@ g++ -v
 llvm-symbolizer --version || true
 
 # Create postgres databases
+if test $CXX = 'clang++'; then
```

Comment:
are we sure that those are the strings?
Maybe use something like this instead?
```
echo "$CXX" | grep -q 'clang++'
if [[ $? -eq 0 ]] ; then
```

Reply:
According to the Travis documentation:

> Testing against two compilers will create (at least) 2 rows in your build matrix. For each row, the Travis CI C++ builder will export the `CXX` env variable to point to either `g++` or `clang++` and `CC` to either `gcc` or `clang`.

so I expect this should work correctly as is.